docs/user/unsupported/mbstyle.rst

```diff
@@ -3,6 +3,11 @@ Mapbox Styles Module
 
 The **gt-mbstyle** module is an unsupported module that provides a parser/encoder to convert between Mapbox Styles and GeoTools style objects. These docs are under active development, along with the module itself.
 
+* Human readable alternative to machine readable XML
+* Compact representation
+* A JSON object with specific root level and nested properties
+* Full access to GeoTools rendering engine not limited by SLD specification
```

Comment:
MBStyle access less of the rendering engine then SLD or YSLD.

If you like we can promote this as a positive, high level configuration of the geotools rendering engine, automatically applying appropriate "vendor options" for legibility (to match the intent of the mbstyle).